### PR TITLE
docs: Fix typos and formatting

### PR DESCRIPTION
**Pull request by jodh-intel**
Correct spelling mistakes and formatting issues.

Fixes: #499.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

File: how-to/how-to-use-virtio-fs-with-kata.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ This document describes how to get Kata Containers to work with virtio-fs.
 
 ## Pre-requisites
 
-* This feature currently requires the host to have hugepages support enabled. Enable this with the `sysctl vm.nr_hugepages=1024` command on the host.
+* This feature currently requires the host to have huge pages support enabled. Enable this with the `sysctl vm.nr_hugepages=1024` command on the host.
```
Comment (grahamwhaley) [inline how-to/how-to-use-virtio-fs-with-kata.md]: hmm, OK, I think I can live with this, but it is `hugepages` I think in every config file and command line etc....

Comment (egernst) [reply how-to/how-to-use-virtio-fs-with-kata.md]: NACK.  Lets add hugepages to the dictionary.

Comment (jodh-intel) [reply how-to/how-to-use-virtio-fs-with-kata.md]: You can't see it yet, but "done" ;)

Comment (jodh-intel) [reply how-to/how-to-use-virtio-fs-with-kata.md]: And branch updated ;)

**Pull request merged**